tests/python/bandit_test_files/skip.py

```diff
@@ -0,0 +1,7 @@
+subprocess.call(["/bin/ls", "-l"])
+subprocess.call(["/bin/ls", "-l"]) #noqa
+subprocess.call(["/bin/ls", "-l"]) # noqa
+subprocess.call(["/bin/ls", "-l"]) # nosec
+subprocess.call(["/bin/ls", "-l"])
+subprocess.call(["/bin/ls", "-l"]) #nosec
+subprocess.call(["/bin/ls", "-l"])
```

Comment:
im afraid these are somehow tested by circleCI. maybe exclude them from py.test or import subprocess so it stops complaining?